rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta

```diff
@@ -17,19 +17,19 @@ func IndexValueObservation: <"Retrieve the values of the supplied index on the s
     // data provider - implementation provides observed value from data source
     inputs:
         observationDate date (1..1)
-        floatingRateOption FloatingRateIndex (1..1)
+        interestRateIndex InterestRateIndex (1..1)
     output:
         observedValue number (1..1)
 
 func IndexValueObservationMultiple: <"Retrieve the values of the supplied index on the specified observation dates.">
     inputs:
         observationDate date (0..*)
-        floatingRateOption FloatingRateIndex (1..1)
+        interestRateIndex InterestRateIndex (1..1)
```

Comment:
Ditto

Reply:
Ditto - Consider moving to `cdm.observable.asset.calculatedrate.func` instead

Reply:
function moved as suggested